src/commands/Lastfm/Rank/AlbumAt.ts

```diff
@@ -31,30 +28,8 @@ export default class AlbumAt extends LastFMBaseCommand<typeof args> {
     rank: new validators.NumberValidator({ whole: true }),
   };
 
-  async run() {
-    const rank = this.parsedArguments.rank;
+  // merged with !albumrank
```

Comment:
```suggestion
```

Reply:
oooops im blind  fixed in 37a6997231672cd7dfdd87b18efec760947ac3f2